Automattic/woocommerce-payments
# Add support for deposit schedule restrictions

Fixes Automattic/woocommerce-payments-server#3009

#### Changes proposed in this Pull Request

This PR adds support for the deposit schedule restriction feature added in Automattic/woocommerce-payments-server#3319. When a deposit schedule restriction is in place, the overview screen shows the hardcoded deposit schedule, and shows deposits are still enabled, but the Payments > Settings page restricts the merchant from altering the deposit schedule. 









#### Testing instructions





* You can use the testing instructions on the server PR to confirm both works.
* Automated tests should pass.

-------------------


**Post merge**
